db/compaction/compaction_job_stats_test.cc

```diff
@@ -571,7 +572,7 @@ CompactionJobStats NewManualCompactionJobStats(
     uint64_t num_input_records, size_t key_size, size_t value_size,
     size_t num_output_files, uint64_t num_output_records,
     double compression_ratio, uint64_t num_records_replaced,
-    bool is_manual = true) {
+    bool is_full = false, bool is_manual = true) {
```

Comment:
Bit of a risky move but...

Reply:
I felt especially comfortable with the change since it's a single-file function. Somewhere else already put "is_full" before "is_manual" so I tried to keep that consistent. :)